services/dataService.js

```diff
@@ -0,0 +1,67 @@
+const productMap = null;
+const categoryMap = null;
+const combinedProductMap = null;
+
+/*
+ * This method loops through the products array and maps them to a variable called productMap
+ */
+const getProducts = () => {
+  const products = require('../src/data/products');
+
+  products.products.forEach(product => {
+    productMap[product.id] = product;
```

Comment:
Better to create a method variable here, and assign that to `productMap` at last.

Reply:
I didn't get this...isn't getProducts() a method variable?

Reply:
Use a new variable inside the function to create the map, and assign that to `productMap` at the end of the forEach loop.